### ISSUE TITLE
why bundle not ask by 'sudo' for install nokogiri 

### ISSUE THREAD

**Issue by kidchenko** [gayanvirajith/harmony#10]
Hi, i am not a ruby specialist, but i ran `bundle install` and i get this:

*An error occurred while installing nokogiri (1.6.6.2), and
Bundler cannot continue.
Make sure that `gem install nokogiri -v '1.6.6.2'` succeeds
before bundling.*

and when i run gem `install nokogiri -v '1.6.6.2'` i get this

*Fetching: nokogiri-1.6.6.2.gem (100%)
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /Library/Ruby/Gems/2.0.0 directory.*

**Comment by gayanvirajith** [comment]
Hi Jose,

Did you `gem install nokogiri -v '1.6.6.2'`?

Thanks

**Comment by lnmunhoz** [comment]
@kidchenko I had the same problem. Use [this tutorial](http://www.nokogiri.org/tutorials/installing_nokogiri.html) maybe can help you.

**Comment by kidchenko** [comment]
@gayanvirajith Yes i tried, It install nokogiri, but even so, **harmony** did not work
@lnmunhoz, tank you for the help, but harmony not work for me :/

**Comment by lvxiaoxin** [comment]
@lnmunhoz Thank you for your link, it is very useful! I finally installed nokogiri.
